modules/core/include/opencv2/core/dualquaternion.inl.hpp

```diff
@@ -387,6 +392,94 @@ DualQuat<T> DualQuat<T>::sclerp(const DualQuat<T> &q0, const DualQuat<T> &q1, co
     return v0 * v0inv1.power(t, QUAT_ASSUME_UNIT);
 }
 
+template <typename T>
+DualQuat<T> DualQuat<T>::dqblend(const DualQuat<T> &q1, const DualQuat<T> &q2, const T t, QuatAssumeType assumeUnit)
+{
+    DualQuat<T> v1(q1), v2(q2);
+    if (!assumeUnit)
+    {
+        v1 = v1.normalize();
+        v2 = v2.normalize();
+    }
+    if (v1.getRotation().dot(v2.getRotation()) < 0)
```

Comment:
Here `QUAT_ASSUME_UNIT` can be used in `getRotation` calls

Reply:
OK